objetos/Aluno.java

```diff
@@ -28,4 +36,25 @@ public int getIdade() {
     public void setIdade(int idade) {
         this.idade = idade;
     }
+
+    public StatusMatricula getStatusMatricula() {
+        return statusMatricula;
+    }
+
+    public boolean setStatusMatricula(String statusMatricula) {
+        try {
+            this.statusMatricula = StatusMatricula.valueOf(statusMatricula);
+            return true;
+        } catch (IllegalArgumentException e) {
+            System.out.println("Status de matrícula invalido.");
+            return false;
+        }
+    }
+
+    public void imprimirOpcoesStatusMatricula() {
+        StatusMatricula[] opcoesStatus = StatusMatricula.values();
+        for (StatusMatricula status : opcoesStatus) {
```

Comment:
O for pode ser alterado para um fori, para exibir números ao lado de cada tipo de status diferente.
Exemplo:

1. ATIVO
2. TRANCADO
3. FORMADO